tox_poetry_installer/hooks.py

```diff
@@ -245,39 +177,18 @@ def tox_testenv_install_deps(venv: ToxVirtualEnv, action: ToxAction) -> Optional
             project_deps = []
             logger.info("Env does not install project package dependencies, skipping")
     except exceptions.ToxPoetryInstallerException as err:
-        venv.status = err.__class__.__name__
+        # tox_env.status = err.__class__.__name__
         logger.error(str(err))
-        return False
+        raise err
     except Exception as err:
-        venv.status = "InternalError"
+        # tox_env.status = "InternalError"
         logger.error(f"Internal plugin error: {err}")
         raise err
 
-    dependencies = utilities.dedupe_packages(
-        dev_deps + group_deps + env_deps + project_deps
-    )
-    if (
-        venv.envconfig.config.option.parallel_install_threads
-        != constants.DEFAULT_INSTALL_THREADS
-    ):
-        parallel_threads = venv.envconfig.config.option.parallel_install_threads
-    else:
-        parallel_threads = (
-            venv.envconfig.config.option.parallelize_locked_install
-            if venv.envconfig.config.option.parallelize_locked_install is not None
-            else constants.DEFAULT_INSTALL_THREADS
-        )
-    log_parallel = f" (using {parallel_threads} threads)" if parallel_threads else ""
-
-    action.setactivity(
-        __about__.__title__,
-        f"Installing {len(dependencies)} dependencies from Poetry lock file{log_parallel}",
```

Comment:
Can you add a logging message before the `installer.install()` call below with a message to this effect? Something like:
```python
logger.info(f"Installing {len(dependencies)} dependencies from Poetry lock file")
```
(no need to keep the parallel threads notice in there)

Reply:
Added the suggested log message.